tipCalculator.js

```diff
@@ -0,0 +1,38 @@
+$('.plus').click(function(){
+    $('#people').val(Number($('#people').val()) + 1)   ;
+});
+
+$('.minus').click(function(){
+    if($('#people').val() == '1'){
+        $('#people').val(1);
+    } else {
+        $('#people').val(Number($('#people').val()) - 1)   ;
```

Comment:
Instead of the `Number` object use `+` to coerce the string into a primitive number.
It's very easy to get tripped up and do `const val = new Number("1")`, at which point `typeof val === "object"` instead of `typeof val === "number"` (!!)